README.md

```diff
@@ -205,8 +206,8 @@ $a->toArray(); // [0 => 'c', 1 => 'a', 2 => 'e', 3 => 'd', 4 => 'b']
 
 ### Converting
 
-Easily converting instance array elements to a simple PHP `array`, `string`,
-`readable string` or `JSON` format:
+Easily convert instance array elements to a simple PHP `array`, `string`,
+`readable string` or JSON format:
```

Comment:
I think we should add quotes to the `JSON` or remove them around `readable string`, because `readable string` isn't  a PHP type.